src/go/cmd/api_view.go

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"io/ioutil"
+)
+
+const (
+	text          = 0
+	newline       = 1
+	whitespace    = 2
+	punctuation   = 3
+	keyword       = 6
+	lineIDMarker  = 5
+	typeName      = 4
+	memberName    = 7
+	stringLiteral = 8
+	literal       = 9
+	comment       = 10
+)
+
+var (
+	reservedNames = map[string]bool{
```

Comment:
I'm guessing `bool` is not really important here, i.e. you just wanted a hash set?  Pattern in Go is to use `struct{}` for the value type instead.

Reply:
Done